CCIP: get status offchain (#1825)
smartcontractkit/documentation: src/content/ccip/tutorials/index.mdx

---
section: ccip
date: Last Modified
title: "CCIP tutorials"
---

> Explore our comprehensive guides to learn cross-chain interoperability using Chainlink's Cross-Chain Interoperability Protocol (CCIP). These tutorials provide step-by-step instructions to understand different patterns you can incorporate into your blockchain projects.

Reviewer: ```suggestion
You can explore several comprehensive guides to learn about cross-chain interoperability using CCIP. These tutorials provide step-by-step instructions to help you understand different patterns that you can incorporate into your blockchain projects.
```
